source/server/server.cc

```diff
@@ -425,6 +425,11 @@ void InstanceImpl::initialize(Network::Address::InstanceConstSharedPtr local_add
                                     messageValidationContext().staticValidationVisitor(), *api_);
   bootstrap_config_update_time_ = time_source_.systemTime();
 
+  if (bootstrap_.has_application_log_format() &&
+      bootstrap_.application_log_format().has_json_format()) {
+    Logger::Registry::setJsonLogFormat(bootstrap_.application_log_format().json_format());
```

Comment:
should probably be added to the config_validation as well.

Reply:
There's a separate server just for config-validation:
https://github.com/envoyproxy/envoy/blob/b978af64641aa46d3b7012ec1d9f3d7d9fd3b307/source/server/config_validation/server.cc#L72

Reply:
gotcha, added. Thanks.